### PR TITLE
Touch ups 3

### PR DESCRIPTION
**Pull request by thomwise**
Just put on the finishing touches. Added the new carousel, events/resources images on home page, and a few growing css animations for links.

### CODE REVIEW

**Review by RobNelson2989 — APPROVED** [review]
Looks great. The code looks really clean. no changes from me

File: functions.php [hunk 1]
```diff
@@ -509,6 +555,7 @@ function namespace_login_style() {
 add_action('widgets_init', 'my_remove_recent_comments_style'); // Remove inline Recent Comment Styles from wp_head()
 add_action('init', 'html5wp_pagination'); // Add our HTML5 Pagination
 add_action( 'login_head', 'namespace_login_style' ); // Adds customization for login screen
+add_action('init', 'event_custom_post_type'); // Adds event custom post type to wp-admin
 
 // Remove Actions
```
Comment (howdycrystalm) [inline functions.php:559]: Could this be removed?

Comment (howdycrystalm) [inline functions.php:559]: Could this be removed?

**Review by howdycrystalm — APPROVED** [review]

File: functions.php (hunk 1 above)
Comment (thomwise) [reply functions.php:559]: No

**Pull request merged**